### DISCUSSION TITLE
Responsive iframe embeds for your web page

### DISCUSSION THREAD

**Discussion by Moonbase59** [AzuraCast/AzuraCast#6812] [Show and tell]
Starting with Rolling Release #[8b15a55](https://github.com/AzuraCast/AzuraCast/commit/8b15a55bad985c3d8eebb0ec2fbff7d7ee2cf7f9) (2023-12-21 23:00), we can now have **responsive iframe embedded widgets** on our websites! Let your users rotate their smartphones/tablets or resize their browser windows, and you’ll always get beautiful iframes with **no scrollbars** and **complete content**.

I added a [Gist](https://gist.github.com/Moonbase59/fbe49e4be4ea9e833f29e94c8526455f) with example code and [Instructions](https://gist.github.com/Moonbase59/fbe49e4be4ea9e833f29e94c8526455f?permalink_comment_id=4803915#gistcomment-4803915) so we all can benefit from this.

[Live example](https://grav1.niteradio.net/wh/iframe-resizer.html): Try rotating your phone/tablet, switch between 10 and 25 results/page to see how it works.

_Note: This is a short-term example, it will go away soon. Use the code in the Gist to try for yourself!_

Thanks @BusterNeece for taking the time to implement this!

**Comment by Heimatradio** [comment]
Thanks a lot @Moonbase59  for this, very useful !!!
Frohe Weihnachten , Merry Christmas , Vrolijk Kerstfeest

**Comment by Moonbase59** [comment]
Thank @BusterNeece for going along with my crazy ideas and giving us all the nice Christmas presents. I just sometimes go and try to see things from a user’s perspective (ok, and almost 50 years of living in IT…).

Merry Christmas to all! Think of those you love.
